examples/step-20/step-20.cc

```diff
@@ -473,16 +473,13 @@ namespace Step20
     // refer to the velocities (a set of <code>dim</code> components starting
     // at component zero) or the pressure (a scalar component located at
     // position <code>dim</code>):
-    const FEValuesExtractors::Vector velocities(0);
-    const FEValuesExtractors::Scalar pressure(dim);
+    const FEValuesExtractors::Vector u(0);
+    const FEValuesExtractors::Scalar p(dim);
```

Comment:
Do the old names make the lines too long?

Reply:
Yes! :smile:

Reply:
I don't think this is a good idea. That's because `u` and `p` are the solution variables, but the extractors are also used to extract components of the *test* functions `v` and `q`. We have also been quite careful to not use single-character variables for anything other than counting indices.

Reply:
I do, in general, dislike 1 character variable names but these don't shadow anything and allow some expressions to fit on one line, so I think they are okay here.

Reply:
But they're misleading because they're not just used for the trial functions but also for the test functions.

I'll add that it seems backward that we give up readability because the tool we use doesn't allow us to write readable code. But that's a separate discussion.

Reply:
The longer version is less beautiful (since we have to split lines) but more readable, so its for the best that we use the longer names.

My style preferences are weak enough that these limitations don't really bother me.